packages/moonstone/CHANGELOG.md

```diff
@@ -23,6 +27,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 ### Fixed
 
 - `moonstone/TimePicker` to show `meridiem` correctly in all locales
+- `moonstone/Scrollable/ScrollButtons` to read out out audio guidance when button down.
```

Comment:
We should not reference non-public components in the CHANGELOG.  Reference the public components affected by the change instead.

Reply:
Oh, I should've removed this line. Forgot to remove after auto-merging. Sorry!